pokemongo_bot/logger.py

```diff
@@ -18,3 +28,112 @@ def log(msg, color=None):
+        super(LogFormatter, self).__init__(fmt, datefmt)
+
+    def get_color(self, record):
+        # We default to white
+        color = LogFormatter.color_hex.get(getattr(record, 'color', None), None) or \
+                self.color_hex.get(record.levelname, LogFormatter.color_hex['white'])
+        return color
+
+    def format(self, record):
+        level = record.levelname
+        color = self.get_color(record)
+        record.msg = '{color}{message}{reset}'.format(color=LogFormatter.COLOR_SEQ % (color), message=record.msg, reset=LogFormatter.RESET_SEQ)
+#        record.msg = '{color}{message}'.format(color='', message=record.msg)
+        result = logging.Formatter.format(self, record)
+        return result
+
+VERBOSE_LEVELV_NUM = 5
+WHITE_LEVELV_NUM = 21
+GREEN_LEVELV_NUM = 22
+YELLOW_LEVELV_NUM = 23
+BLUE_LEVELV_NUM = 24
+CYAN_LEVELV_NUM = 25
+RED_LEVELV_NUM = 26
+def custom_logging_level_handler(self, level, message, args, kwargs):
+    if self.isEnabledFor(level):
+        self._log(level, message, args, **kwargs)
+def verbose(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, VERBOSE_LEVELV_NUM, message, args, kwargs)
+def white(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, WHITE_LEVELV_NUM, message, args, kwargs)
+def green(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, GREEN_LEVELV_NUM, message, args, kwargs)
+def yellow(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, YELLOW_LEVELV_NUM, message, args, kwargs)
+def blue(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, BLUE_LEVELV_NUM, message, args, kwargs)
+def cyan(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, CYAN_LEVELV_NUM, message, args, kwargs)
+def red(self, message, *args, **kwargs):
+    return custom_logging_level_handler(self, RED_LEVELV_NUM, message, args, kwargs)
+def colorized(self, level, message, color, *args, **kwargs):
+    extra = {'color': color}
+    kwargs['extra'] = extra
+    return custom_logging_level_handler(self, level, message, args, kwargs)
+
+logging.Logger.verbose = verbose
+logging.addLevelName(VERBOSE_LEVELV_NUM, 'verbose')
+logging.Logger.white = white
+logging.addLevelName(WHITE_LEVELV_NUM, 'white')
+logging.Logger.green = green
+logging.addLevelName(GREEN_LEVELV_NUM, 'green')
+logging.Logger.yellow = yellow
+logging.addLevelName(YELLOW_LEVELV_NUM, 'yellow')
+logging.Logger.blue = blue
+logging.addLevelName(BLUE_LEVELV_NUM, 'blue')
+logging.Logger.cyan = cyan
+logging.addLevelName(CYAN_LEVELV_NUM, 'cyan')
+logging.Logger.red = red
+logging.addLevelName(RED_LEVELV_NUM, 'red')
+logging.Logger.colorized = colorized
+
+
+root = logging.getLogger()
+if lcd:
+    root.addHandler(logger.LcdHandler())
+
+def init_logger():
+    rootLogger = logging.getLogger()
+    for handler in rootLogger.handlers:
+        handler.setFormatter(LogFormatter())
+
+
```

Comment:
Please, removing anything not related to your PR from this file, like the LcdHandler.